wandb/data_types.py

```diff
@@ -881,7 +884,17 @@ def __init__(self, data_or_path, sample_rate=None, caption=None):
         self._caption = caption
 
         if isinstance(data_or_path, six.string_types):
-            self._set_file(data_or_path, is_tmp=False)
+            if Audio.path_is_reference(data_or_path):
+                if sample_rate is None:
+                    raise ValueError(
+                        'Argument "sample_rate" is required when instantiating wandb.Audio with a reference path.'
+                    )
+                
```

Comment:
Why is sample_rate required when we're using a reference path?

Reply:
I thought that we always needed the sample rate prior to loading the file to make Wavesurfer work in the browser, so you'd need to provide it manually to avoid resolving the reference at the time it's added. Looking at this again, though, you may be right -- it seems like it isn't actually used. I'm gonna try taking it out and see if everything still works.

Reply:
follow-up: This definitely isn't necessary -- I've removed it. Thanks for catching it!